services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from datetime import time, date
+
+
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: time,

```

Comment:
The `movie_show_time` parameter should be a `datetime` object instead of `time`, as the `show_time` field in the `MovieSession` model is a `DateTimeField`. Update the parameter type and ensure the function handles `datetime` objects correctly.